.github/ISSUE_TEMPLATE/sentence_issue.yaml

```diff
@@ -0,0 +1,130 @@
+
+        [fr]: https://community.home-assistant.io/c/feature-requests
+  - type: dropdown
+    validations:
+      required: true
+    attributes:
+      label: Which language are you using?
+      description: >
+        Select the language code of the language in which you are using the Assist function.
+      options:
+        - ar
+        - bg
+        - bn
+        - ca
+        - cs
+        - da
+        - de
+        - de-CH
+        - el
+        - en
+        - es
+        - fa
+        - fi
+        - gl
+        - he
+        - hi
+        - hr
+        - hu
+        - id
+        - is
+        - it
+        - ka
+        - kn
+        - lb
+        - lt
+        - lv
+        - ml
+        - mn
+        - ms
+        - nb
+        - nl
+        - pl
+        - ro
+        - ru
+        - sk
+        - sl
+        - sr
+        - sv
+        - sw
+        - te
+        - tr
+        - uk
+        - ur
+        - vi
+        - zh-cn
+        - zh-hk
+        - zh-tw
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: The sentence which is not working
+      description: >-
+        Provide the sentence or sentences which are not working
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: Expected result
+      description: >-
+        What was your expected result of the above sentence(s)?
+  - type: markdown
+    attributes:
+      value: |
+        ## Environment
+  - type: input
+    id: version
+    validations:
+      required: true
+    attributes:
+      label: What version of Home Assistant Core has the issue?
+      placeholder: core-
+      description: >
+        Can be found in: [Settings -> About](https://my.home-assistant.io/redirect/info/).
+
+        [![Open your Home Assistant instance and show your Home Assistant version information.](https://my.home-assistant.io/badges/info.svg)](https://my.home-assistant.io/redirect/info/)
+  - type: input
+    attributes:
+      label: What was the last working version of Home Assistant Core?
+      placeholder: core-
+      description: >
+        If known, otherwise leave blank.
+  - type: dropdown
+    validations:
+      required: true
+    attributes:
+      label: What type of installation are you running?
+      description: >
+        Can be found in: [Settings -> System-> Repairs -> Three Dots in Upper Right -> System information](https://my.home-assistant.io/redirect/system_health/).
+
+        [![Open your Home Assistant instance and show health information about your system.](https://my.home-assistant.io/badges/system_health.svg)](https://my.home-assistant.io/redirect/system_health/)
+      options:
+        - Home Assistant OS
+        - Home Assistant Container
+        - Home Assistant Supervised
+        - Home Assistant Core
+
+  - type: markdown
+    attributes:
+      value: |
+        # Details
```

Comment:
let's leave this out maybe? it's sentence that's not working, and we already have additional info